Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterEditor/Vector3GizmoParameterEditor.cpp

```diff
@@ -163,6 +156,7 @@ namespace EMStudio
             MCore::AttributeVector3* typedAttribute = static_cast<MCore::AttributeVector3*>(attribute);
             typedAttribute->SetValue(m_currentValue);
         }
+        m_translationManipulators.SetLocalPosition(m_currentValue);
```

Comment:
update the localPosition here because when you set the property manually the manipulator does not update its position.